LandSandBoat/server
# [lua] [core] Adjust intimidate/is paralyzed melee/JA messages to match retail, handle untargetable entities + proper super jump implementation

**_I affirm:_**

## What does this pull request do?
Adjust intimidate/is paralyzed melee/JA messages to match retail

* Fixed duplicate "is paralyzed" messages by sending it properly as an action with the right param/animation
* Move claim on auto attack to match retail jank where you can still claim a monster with an auto attack that is paralyzed
* Move auto attack timer to above para/intimidate checks for trust timer
* Add intimidation check to abilities
* Add super jump untargetable implementation
* Add various untargetable checks to states

## Steps to test these changes



`!setmod paralyze 100` on a target and additionally a player, see it emit one "Is Paralyzed" message per action
`!changejob BST 99`, have a non-beastmen mob attack you, see a single "x intimidated by" message
`!setmod humanoid_killer 100` on a mob, attack it and see constant intimidates on spells/melee attacks
Use super jump and see the mob lose target on you during a spell cast or post TP move ready
Super jump "untargetable" state is testable with `!exec target:untargetableAndUnactionable(5000)` for 5 seconds of being untargetable
